homeassistant/components/hdmi_cec/__init__.py

```diff
@@ -201,6 +240,88 @@ def _async_initialized_callback(*_: Any):
 
     hdmi_network.set_initialized_callback(_async_initialized_callback)
 
+    @callback
+    def handle_cec_command(command: CecCommand):
```

Comment:
Hmm, I think there's a lot of usecase logic in here. It feels like it would make more sense to have a `cec_cmd_received` bus event or something similar, and have the `opcode/cmd` & `att` as part of the context. Then anyone can implement their own logic in automations.